src/HTSPDemuxer.cpp

```diff
@@ -309,13 +309,8 @@ bool CHTSPDemuxer::IsRealTimeStream() const
    * we want the calculation to be consistent */
   CLockObject lock(m_mutex);
 
-  if (m_timeshiftStatus.shift == 0)
-    return true;
-
-  if (m_timeshiftStatus.start - m_timeshiftStatus.shift < 10)
-    return true;
-
-  return false;
+  /* Handle as real time when reading close to the EOF (10000000�s - 10s) */
+  return (m_timeshiftStatus.shift < 10000000);
```

Comment:
@ksooo Now pvr.hts should behave the same as pvr.vnsi. When a stream is "real time" kodi applies an other A/V sync method if I'm right. Not using this method can lead to A/V sync issues when timeshifting close to the live point. Also tvheadends "shift" is equal to "end time - play time" in vdr.
https://github.com/kodi-pvr/pvr.vdr.vnsi/blob/Krypton/src/client.cpp#L870